qubes-rpc/admin.vm.TerminalDispVM

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash --
+
+set -eu
+sock_name="$(mktemp -u XXXXXX)"
+if [[ "$sock_name" =~ [^a-zA-Z0-9] ]]; then
+  echo "Invalid internal sock name provided."
+  exit 1
+fi
+sock="/var/run/qubes/$sock_name.terminal.sock"
+
+xterm -geometry 80x24 -e /bin/sh -c "
+until [ -S $sock ]; do sleep 1; done || true
```

Comment:
I'd put smaller sleep here. One may want to catch early boot...
Also, it may be good to add echo with a message how to exit from this.
But on the other hand, in this case closing the window should be enough.